src/lib.rs

```diff
@@ -404,6 +407,56 @@ mod command_tests {
     }
 }
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub enum TryFromCecLogMessageError {
+    MessageParseError,
+    LogLevelParseError,
+    UnknownLogLevel,
+}
+
+#[derive(Clone)]
+pub struct CecLogMessage {
+    #[doc = "< the actual message, valid until returning from the log callback"]
+    pub message: String,
+    #[doc = "< log level of the message"]
+    pub level: CecLogLevel,
+    #[doc = "< the timestamp of this message"]
+    pub time: i64,
```

Comment:
```suggestion
    #[doc = "duration since connection was established"]
    pub time: Duration,
```

The original docs are quite poor on this one.

This seems to be actually time of the log message since connection start, see

https://github.com/Pulse-Eight/libcec/blob/76551ea1dd9a55f0ce1533e440dc12dbc594f7ba/src/libcec/LibCEC.cpp#L405

and 

https://github.com/Pulse-Eight/libcec/blob/76551ea1dd9a55f0ce1533e440dc12dbc594f7ba/src/dotnetlib/CecSharpTypes.h#L1522

I would propose to use `Duration` here, and improve the

Reply:
Changed to using Duration.